### PR TITLE
Some small fixes

### PR DESCRIPTION
**Pull request by ellert**
* Remove man pages for non-existing binaries
  They might have existed in the past: g2rootold genmap proofserva roota

* Install generated man pages in the correct subdirectory (man/man1)

* Use the correct library name in the plugin definitions

* Remove obsolete plugin definition
  libPeac was removed in version 5.34.01

* Simplify rule
  No need to exclude files that no longer exists

* TWebVirtualX was removed - remove plugin definition using it
  See commit a07af0afbe2e26237bf8ca059abc34255378a070

* Don't download input file if it already exists
  This allows running the tests with pre-fetched file in environments without network - like e.g. the Fedora package build server

* Python 3 compatibility

### CODE REVIEW

**Comment by amadio** [comment]
@phsft-bot build

File: cmake/modules/RootNewMacros.cmake [hunk 1]
```diff
@@ -1540,5 +1540,5 @@ function(generateManual name input output)
       ${PYTHON_EXECUTABLE} -B ${CMAKE_SOURCE_DIR}/build/misc/argparse2help.py ${input} ${output}
   )
 
-  install(FILES ${output} DESTINATION ${CMAKE_INSTALL_MANDIR})
+  install(FILES ${output} DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
```
Comment (amadio) [inline cmake/modules/RootNewMacros.cmake]: Thanks for this! I had noticed before, but didn't get around to fix it yet.

File: tutorials/dataframe/df014_CSVDataSource.C [hunk 2]
```diff
@@ -23,7 +23,8 @@ int df014_CSVDataSource()
    // The types of the columns will be automatically inferred.
    auto fileNameUrl = "http://root.cern.ch/files/tutorials/df014_CsvDataSource_MuRun2010B.csv";
    auto fileName = "df014_CsvDataSource_MuRun2010B_cpp.csv";
-   TFile::Cp(fileNameUrl, fileName);
+   if(gSystem->AccessPathName(fileName))
```
Comment (amadio) [inline tutorials/dataframe/df014_CSVDataSource.C]: Shouldn't this be `if(!gSystem->AccessPathName(fileName))`? That is, copy over if it cannot be accessed?

Comment (Axel-Naumann) [reply tutorials/dataframe/df014_CSVDataSource.C]: From the doc: `Returns FALSE if one can access a file using the specified access mode`...

Comment (amadio) [reply tutorials/dataframe/df014_CSVDataSource.C]: That's terrible UX, I must say.

Comment (Axel-Naumann) [reply tutorials/dataframe/df014_CSVDataSource.C]: O yeah. But it's 25 years old, and inverting the return value is among the worst things one can do :-)

**Comment by amadio** [comment]
@ellert Could you please fork also roottest? There is a bug in the infrastructure that will fail Windows builds if you don't have a fork of roottest as well as root... Thanks!

**Comment by ellert** [comment]
> @ellert Could you please fork also roottest?

Done

**Comment by amadio** [comment]
Thanks!

**Comment by amadio** [comment]
@phsft-bot build just on windows10/default

File: tutorials/dataframe/df014_CSVDataSource.C (hunk 2 above)
Comment (pcanal) [reply tutorials/dataframe/df014_CSVDataSource.C]: It is even old than that :) ... see ```man access```

**Pull request merged**